### PR TITLE
Initial setup, added routing, sidenav navigation, both Login and Register forms, 

### PR DESCRIPTION
**Pull request by ironkajman**
- Initial setup
- Routing
- Login component
- Register component
- Sidenav navigation

### CODE REVIEW

**Comment by stepaniukm** [comment]
Written in Angular, so the score can only be this one -> 💩

**Review by stepaniukm — CHANGES_REQUESTED** [review]
Rewrite in some modern framework

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import {LoginComponent} from './login/login.component';
+import {RegisterComponent} from './register/register.component';
 
-const routes: Routes = [];
+const routes: Routes = [
```
Comment (stepaniukm) [inline src/app/app-routing.module.ts:6]: use more useful `satisfies` operator

Comment (ironkajman) [reply src/app/app-routing.module.ts:6]: Irrelevant change

File: src/app/login/login.component.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+
+export class LoginComponent implements OnInit{
+
+  loginForm: FormGroup
+
+  ngOnInit() {
+    this.loginForm = new FormGroup({
```
Comment (Deeadline) [inline src/app/login/login.component.ts]: In angular 14+ where we have typed forms it is good to create `FormGroup` inline. So you can access controls like
`this.loginForm.controls.login` and typescript + angular know it is safe prop.

Comment (ironkajman) [reply src/app/login/login.component.ts]: done :)

**Pull request merged**